### PR TITLE
Radio control registry

### PR DESCRIPTION
**Pull request by sr5434**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
I provided an internal service(the `RadioControlRegistry`) in root and cleaned up the code that uses that service.

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by JeanMeche** [comment]
I believe this change is not wanted. `RadioControlRegistry` is for internal use only and should not be provided in root.

**Comment by sr5434** [comment]
`RadioControlRegistry` is an internal-only service and providing it in root wouldn't make it public. However, if it is provided in root, some internal code that references it can be cleaned up.

**Comment by dylhunn** [comment]
Is there any other benefit besides removing the `RadioControlRegistry` from the imports array?

File: packages/forms/src/directives.ts [hunk 1]
```diff
@@ -78,7 +77,7 @@ export const REACTIVE_DRIVEN_DIRECTIVES: Type<any>[] =
  */
 @NgModule({
   declarations: SHARED_FORM_DIRECTIVES,
-  imports: [RadioControlRegistryModule],
```
Comment (JoostK) [inline packages/forms/src/directives.ts:81]: You could remove all of `RadioControlRegistryModule`, it has gone unused with these changes.

Comment (sr5434) [reply packages/forms/src/directives.ts:81]: Isn’t this the only place where it it imported?

**Comment by sr5434** [comment]
> Is there any other benefit besides removing the `RadioControlRegistry` from the imports array?

No, not really. This commit is simply meant to simplify a little bit of code.

**Comment by sr5434** [comment]
Willl this be merged?

**Comment by dylhunn** [comment]
This passed g3 presubmit. We should be good to merge it.

**Review by dylhunn — APPROVED** [review]
reviewed-for: fw-forms

**Comment by dylhunn** [comment]
@sr5434 You also might be able to delete the module entirely, as Joost pointed out. But we can merge as-is also.

**Comment by sr5434** [comment]
Why is CircleCI not working?

**Comment by JeanMeche** [comment]
You need to rebase on main !

**Comment by AndrewKushnir** [comment]
@sr5434 FYI I've just pushed a rebase to rerun the CI.

**Comment by AndrewKushnir** [comment]
> So now I don’t have to rebase, just what you listed, right?

Yes, you can just fix the mentioned issues without an extra rebase.

**Comment by sr5434** [comment]
Thanks. I will get to work on those changes ASAP.

**Comment by sr5434** [comment]
I think the issue closed when I force pushed to my branch.

**Comment by JeanMeche** [comment]
Yeah there is an issue with your push, the PR has a delta of 5000+ files

**Comment by sr5434** [comment]
Is it possible to fix? When I looked at the list of file changes in VS code, it only showed 3 or 4 files.

**Comment by sr5434** [comment]
GitHub is not letting me reopen the pull request. Can anyone else reopen it?

**Comment by JeanMeche** [comment]
You will need to reopen a new one !

**Comment by sr5434** [comment]
Ok, I will open a new one.

**Comment by sr5434** [comment]
Reopened as #50037

**Pull request closed without merging**